kolibri/utils/lru_cache.py

```diff
@@ -0,0 +1,177 @@
+"""
```

Comment:
All of this needs no review, it's been added because it needs to be accessed in setup.py BEFORE any requirement is installed (as for instance Django, which is where it originates from)